scanner/build.gradle.kts

```diff
@@ -20,23 +20,11 @@
 
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
-val exposedVersion: String by project
-val hikariVersion: String by project
-val jacksonVersion: String by project
-val kotlinxCoroutinesVersion: String by project
-val kotlinxSerializationVersion: String by project
-val mockkVersion: String by project
-val postgresVersion: String by project
-val retrofitKotlinxSerializationConverterVersion: String by project
-val retrofitVersion: String by project
-val scanossVersion: String by project
-val sw360ClientVersion: String by project
-val wiremockVersion: String by project
-
 val askalonoVersion: String by project
 val boyterLcVersion: String by project
 val licenseeVersion: String by project
 val scancodeVersion: String by project
+val scanossVersion: String by project
```

Comment:
Couldn't you get this version from the `libs` object like you did in some places in the previous commit? To avoid that the version is declared in two places.